app/licenses.yml

```diff
@@ -154,7 +154,3 @@
   name: debot
   copyrightHolder: Tomoaki Imai
   license: The Apache License, Version 2.0
-- artifact: jp.wasabeef:takt:+
```

Comment:
It's okay now!
I'll ask to @gfx how we should handle the case of `debugCompile`.